tests/integration/Storefront/Theme/ThemeTest.php

```diff
@@ -248,10 +241,12 @@ public function testChildThemeConfigStructuredFieldsInheritance(): void
         $childTheme = $this->themeRepository->search($criteria, $this->context)->getEntities()->first();
         static::assertNotNull($childTheme);
 
+        /** @var string $technicalName */
```

Comment:
please do not use var annotations. Especially in tests you can simply assert the type. also see https://github.com/shopware/shopware/blob/trunk/coding-guidelines/core/writing-code-for-static-analysis.md